src/functions/utils.php

```diff
@@ -638,4 +638,36 @@ function tribe_register_rest_route( $namespace, $route, $args = array(), $overri
 		$args = apply_filters( 'tribe_register_rest_route_args', $args, $namespace, $route, $override );
 		return register_rest_route( $namespace, $route, $args, $override );
 	}
+}
+
+if ( ! function_exists( 'tribe_get_request_vars' ) ) {
+	/**
+	 * Returns the sanitized version of the `$_REQUEST` super-global array.
```

Comment:
We should probably note that this is cached and reused. Any manual changes to $_REQUEST will not be reflected.

Reply:
This is a good idea, thanks

Reply:
I've added a note and added support for a `$refresh` parameter to allow refreshing the cache.